runtime/j9vm/j7vmi.c

```diff
@@ -2328,30 +2328,6 @@ JVM_SuspendThread(jint arg0, jint arg1)
 }
 #endif /* JAVA_SPEC_VERSION < 20 */
 
-/* NOTE this is required by JDK15+ jdk.internal.loader.NativeLibraries.unload().
- */
-#if JAVA_SPEC_VERSION >= 15
-void JNICALL JVM_UnloadLibrary(void *handle)
-#else /* JAVA_SPEC_VERSION >= 15 */
-jobject JNICALL JVM_UnloadLibrary(jint arg0)
-#endif /* JAVA_SPEC_VERSION >= 15 */
-{
-#if JAVA_SPEC_VERSION >= 15
-	Trc_SC_UnloadLibrary_Entry(handle);
-#if defined(WIN32)
-	FreeLibrary((HMODULE)handle);
-#elif defined(J9UNIX) || defined(J9ZOS390) /* defined(WIN32) */
-	dlclose(handle);
-#else /* defined(WIN32) */
-#error "Please implement J7vmi.c:JVM_UnloadLibrary(void *handle)"
-#endif /* defined(WIN32) */
-#else /* JAVA_SPEC_VERSION >= 15 */
-	assert(!"JVM_UnloadLibrary() stubbed!");
-	return NULL;
-#endif /* JAVA_SPEC_VERSION >= 15 */
-}
```

Comment:
This seems to have been moved unchanged to `jvm.c` - why not leave it here?

Reply:
Access to the BFUjavaVM variable, but it can be declared extern in this file.

Reply:
I don't have a strong preference; I'm fine with moving this function to `jvm.c`.